pom.xml

```diff
@@ -345,7 +345,7 @@
         <kotlin-version>1.9.22</kotlin-version>
         <kotlinpoet-version>1.15.3</kotlinpoet-version>
         <kubernetes-client-version>6.10.0</kubernetes-client-version>
-        <kubernetes-model-version>6.9.2</kubernetes-model-version>
+        <kubernetes-model-version>6.10.0</kubernetes-model-version>
```

Comment:
In Camel, the property kubernetes-model-version is not in use, and as a result, all kubernetes artifacts point to version `6.10.0`

Reply:
You are partially right, the camel project doesn't use the property `kubernetes-model-version` but it uses `kubernetes-client-version` so you should use it even for the model